source_code/aux_mcu_v4/src/COMMS/comms_aux_mcu_defines.h

```diff
@@ -14,5 +14,20 @@
 // HID payload size
 #define HID_PAYLOAD_SIZE            64
 
+//FIDO2 related lengths/sizes
+#define FIDO2_RPID_LEN 252
+#define FIDO2_USER_ID_LEN 65
+#define FIDO2_USER_NAME_LEN 65
+#define FIDO2_DISPLAY_NAME_LEN 32
```

Comment:
in your struct: 

uint8_t display_name[FIDO2_DISPLAY_NAME_LEN];  //65

Reply:
fido spec: This identifier is intended for display. [=Authenticators=] MUST accept and store a 64 byte minimum length
        for a name members's value. Authenticators MAY truncate a
        name member's value to a length equal to or greater than 64 bytes.

Reply:
I know where I got that value from (Solo impl):
"#define DISPLAY_NAME_LIMIT          32  // Must be minimum of 64 bytes but can be more."
Fixed.